docs/collect-forms.rst

```diff
@@ -48,6 +50,44 @@ The :guilabel:`Drafts` list shows all drafts by the name that the :ref:`form def
   :alt: The Drafts screen. Several drafts are listed by name.
   :class: device-screen-vertical
 
+.. _finalizing_drafts:
+
+Finalizing drafts
+==================
+
+.. note::
+
+  In Collect versions prior to v2023.3, it was possible to edit finalized forms. If your workflow involves new information that can be discovered at any time, consider configuring Collect to hide the :guilabel:`Finalize` or :guilabel:`Send` button from the form end screen (see the form entry access control section of :ref:`protected settings <admin-settings>`) and using :ref:`bulk finalization <bulk-finalizing-drafts>`.
```

Comment:
If your workflow involves adding or removing data in the form before it has to be sent.